util/generate-network-configurations.py

```diff
@@ -0,0 +1,161 @@
+from troposphere import Equals, GetAtt, If, Not, Output, Parameter, Ref, Template
+from troposphere.ec2 import (
+    EIP,
+    InternetGateway,
+    NatGateway,
+    Route,
+    RouteTable,
+    Subnet,
+    SubnetRouteTableAssociation,
+    VPCGatewayAttachment,
+)
+
+
+class NetworkHandler:
+    def __init__(self):
+        self.template = Template()
+
+    @staticmethod
+    def _add_name(resource_name):
+        return [{"Key": "Name", "Value": f"pcluster-{resource_name}"}]
+
+    def _add_parameter(self, name, description, expected_input):
```

Comment:
s/expected_input/type?

Reply:
I am sorry but I don't understand what you mean